### PR TITLE
clean typo

### PR DESCRIPTION
**Pull request by Ray-cmd**
clean typo

### CODE REVIEW

**Comment by FizzYxWizzY** [comment]
GG!

**Comment by Ray-cmd** [comment]
ggwp

**Comment by FizzYxWizzY** [comment]
LGTM

File: docs/parser.md [hunk 1]
```diff
@@ -220,3 +220,4 @@ int main(void)
 	dyntree_free(root);
 	return (0);
 }
+```
```
Comment (Ray-cmd) [inline docs/parser.md:223]: what a play

**Pull request merged**